install/Makefile

```diff
@@ -38,32 +38,26 @@ crd_install_helm2:
 crd_install_helm3:
 	$(CP) ./cluster-operator/043-Crd-kafkatopic.yaml ./topic-operator/04-Crd-kafkatopic.yaml
 	$(CP) ./cluster-operator/044-Crd-kafkauser.yaml ./user-operator/04-Crd-kafkauser.yaml
-	$(CP) ./cluster-operator/040-Crd-kafka.yaml ../helm-charts/helm3/strimzi-kafka-operator/templates/040-Crd-kafka.yaml
-	$(CP) ./cluster-operator/041-Crd-kafkaconnect.yaml ../helm-charts/helm3/strimzi-kafka-operator/templates/041-Crd-kafkaconnect.yaml
-	$(CP) ./cluster-operator/042-Crd-kafkaconnects2i.yaml ../helm-charts/helm3/strimzi-kafka-operator/templates/042-Crd-kafkaconnects2i.yaml
-	$(CP) ./cluster-operator/043-Crd-kafkatopic.yaml ../helm-charts/helm3/strimzi-kafka-operator/templates/043-Crd-kafkatopic.yaml
-	$(CP) ./cluster-operator/044-Crd-kafkauser.yaml ../helm-charts/helm3/strimzi-kafka-operator/templates/044-Crd-kafkauser.yaml
-	$(CP) ./cluster-operator/045-Crd-kafkamirrormaker.yaml ../helm-charts/helm3/strimzi-kafka-operator/templates/045-Crd-kafkamirrormaker.yaml
-	$(CP) ./cluster-operator/046-Crd-kafkabridge.yaml ../helm-charts/helm3/strimzi-kafka-operator/templates/046-Crd-kafkabridge.yaml
-	$(CP) ./cluster-operator/047-Crd-kafkaconnector.yaml ../helm-charts/helm3/strimzi-kafka-operator/templates/047-Crd-kafkaconnector.yaml
-	$(CP) ./cluster-operator/048-Crd-kafkamirrormaker2.yaml ../helm-charts/helm3/strimzi-kafka-operator/templates/048-Crd-kafkamirrormaker2.yaml
-	$(CP) ./cluster-operator/049-Crd-kafkarebalance.yaml ../helm-charts/helm3/strimzi-kafka-operator/templates/049-Crd-kafkarebalance.yaml
-	oldpath=`pwd` && cd ../helm-charts/helm3/strimzi-kafka-operator/templates/ &&  $(FIND) . -type f -name '04*-Crd-*.yaml' -exec yq write -i {} metadata.labels.app "{{ template \"strimzi.name\" . }}" \; && cd $$oldpath
```

Comment:
Since the CRDs no longer contain templates, we're able to remove several usage of `yq` to modify the yml.